vitest.config.e2e.ts

```diff
@@ -0,0 +1,11 @@
+import { configDefaults, defineConfig } from "vitest/dist/config";
+
+export default defineConfig({
+  test: {
+    singleThread: true,
```

Comment:
is single thread necessary? works fine locally when parallelized

Reply:
single thread is necessary so that you can have tests wait between executes.

To solve the `replacement` underpriced issue, you would have to have tests wait for one another w/ a sleep

Another option is, in the beforeAll step, you would call `getNonce`. Then for each test you add an offset (0-indexed) that's specific for that test.